src/backend/joanie/core/models/certifications.py

```diff
@@ -54,38 +55,41 @@ class Certificate(models.Model):
         # disable=all is necessary to avoid an AstroidImportError because of our models structure
         # Astroid is looking for a module models.py that does not exist
         "core.Order",  # pylint: disable=all
+        related_name=_("issued_certificate"),
```

Comment:
oups, related_name should not be translated!

Reply:
Oh damn ! Good catch 🙏